vendor/bat-native-ads/src/bat/ads/internal/ads/serving/targeting/models/behavioral/bandits/epsilon_greedy_bandit_model.cc

```diff
@@ -149,7 +149,7 @@ SegmentList ExploitSegments(const EpsilonGreedyBanditArmMap& arms) {
   const ArmBucketMap unsorted_buckets = BucketSortArms(ToArmList(arms));
   const ArmBucketList sorted_buckets = GetSortedBuckets(unsorted_buckets);
   const ArmList top_arms = GetTopArms(sorted_buckets, kTopArmCount);
-  const SegmentList segments = ToSegmentList(top_arms);
+  SegmentList segments = ToSegmentList(top_arms);
```

Comment:
why do we remove const here?

Reply:
const-ness of segments prevented an automative move

Reply:
Where it will be moved?

Reply:
const rvalues trigger copy constructor